Sources/Testing/Attachments/Attachable.swift

```diff
@@ -64,6 +64,22 @@ public protocol Attachable: ~Copyable {
   /// would not be idiomatic for the buffer to contain a textual description of
   /// the image.
   borrowing func withUnsafeBufferPointer<R>(for attachment: borrowing Attachment<Self>, _ body: (UnsafeRawBufferPointer) throws -> R) throws -> R
+
+  /// Generate a preferred name for the given attachment.
+  ///
+  /// - Parameters:
+  ///   - suggestedName: A suggested name to use as the basis of the preferred
+  ///     name. This string was provided by the developer when they initialized
+  ///     `attachment`.
+  ///   - attachment: The attachment that needs to be named.
```

Comment:
Want to flip the order of the parameters to match the signature?

Reply:
Oops, yes, I had moved stuff around after talking to some cow-orkers.